src/lib/logger/langchain/utils.ts

```
@@ -0,0 +1,412 @@
+			message: error,
+			type: "Unknown",
+		};
+	} else {
+		return {
+			message: "An unknown error occurred",
+			type: "Unknown",
+		};
+	}
+}
+
+export function maybeParseJSON(input: string = "") {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)